tests/dags/providers/provider_api_scripts/test_phylopic.py

```diff
@@ -1,33 +1,42 @@
 import json
 import logging
-import os
 from functools import partial
+from pathlib import Path
 from unittest.mock import patch
 
 import pytest
 from common.licenses import get_license_info
 from providers.provider_api_scripts import phylopic as pp
 
 
-RESOURCES = os.path.join(
-    os.path.abspath(os.path.dirname(__file__)), "resources/phylopic"
-)
+RESOURCES = Path(__file__).parent.resolve() / "resources/phylopic"
 
 logging.basicConfig(
     format="%(asctime)s - %(name)s - %(levelname)s:  %(message)s",
     level=logging.DEBUG,
 )
 
 
+@pytest.fixture
+def image_data():
+    IMAGE_DATA_EXAMPLE = RESOURCES / "correct_meta_data_example.json"
+    with open(IMAGE_DATA_EXAMPLE) as f:
+        yield json.load(f)
```

Comment:
Can this fixture be defined using the `get_json` helper?

```suggestion
    yield get_json("correct_meta_data_example.json")
```